pxtlib/emitter/service.ts

```diff
@@ -695,7 +695,7 @@ namespace ts.pxtc.service {
     let lastBlocksInfo: BlocksInfo;
     let lastFuse: Fuse;
     let builtinItems: SearchInfo[];
-    let tbSubset: Map<boolean>;
+    let tbSubset: pxt.Map<boolean>;
```

Comment:
Shall address differently. Not related to this change. We have MapLike & pxt.Map in our codebase